Fushigi/ui/widgets/CourseScene.cs

```diff
@@ -2658,22 +2748,19 @@ private void SelectLayerToAdd()
                 {
                     for (var i = 0; i < fileteredLayers.Count; i++)
                     {
-                        var layer = i < 2 ? 
-                            fileteredLayers[i]+$" ({mLayersVisibility.Count(x => Regex.Replace(x.Key, @"\\d+", "").Contains(fileteredLayers[i]))}/10)":
-                            fileteredLayers[i];
+                        layerCount = mLayersVisibility.Count(x => RxLetters.Replace(x.Key, "") == fileteredLayers[i]);
```

Comment:
Simply using StartsWith instead of removing all numbers with a Regex makes a lot more sense and should also be a lot faster